### ISSUE TITLE
SSR Readout

### ISSUE THREAD

**Issue by kucerazdenek** [petrleocompel/hass-wattrouter-mx#2]
@petrleocompel , z pohledu HA jsem spíš BFU, tudíž jsem odkázán na to co si kde přečtu, naimplementuju a pak to případně upravím.
Chtěl jsem poprosit, bylo by možné do integrace přidat čtení sensorů SSR? Zkoušel jsem to přes Multiscrape, nicméně mi to funguje velice podivně a nevím proč (zejména refresh interval). Zajímalo by mě kolik energie z přetoků nacpu do ohřevu TUV tou patronou v tepelném čerpadle. Jestli správně chápu wattconfig rozhraní, jedná se o následující hodnoty na obrázku, přičemž předpokládám, že load power je okamžitý předpokládaný/odhadovaný příkon per fáze a supplied energy je energie (u té se pravděpodobně counter resetuje na denní bázi) i když u té si nejsem jistý co vlastně ukazuje, protože dneska patrona ještě neběžela a i přesto tam nějaké hodnoty jsou. Možná by mi stačil jen load power a  spotřebu v kWh si z toho přes Riemannův integrál udělám sám...

![image](https://github.com/petrleocompel/hass-wattrouter-mx/assets/121700009/0a015369-ca67-4c22-a912-91160a798f3a)

**Comment by petrleocompel** [comment]
"feature request" jsou pořád issue. (Issue není nutně chyba).

Jediný problém je vzorkování. Interval jak často budeme sbírat data. Zkouknu jak Wattrouter rozhraní nyní pracuje. Pokud se jedná o opakovaný request nebude problém, pokud se bude jednat o kontinuální stream bude to trochu horší. 

Jinak v tom nevidím problém v XML jsem asi hodnoty viděl.

**Comment by kucerazdenek** [comment]
Ideální by bylo pokud by byl interval nastavitelný v sekundách. Zadával by se při prvotní konfiguraci integrace stejně jako nyní URL, jméno a heslo

**Comment by petrleocompel** [comment]
Ano, ale problém bude na straně Watterouter. Například konfiguraci nelze
zapsat "rozděleně". I nyní ovládání relé mění konfiguraci a proto existují
skupiny pro ovládání. Jelikož Wattrouter nedokáže zpracovat konfiguraci tak
rychle. Uvidím ale jinak souhlas.

čt 1. 6. 2023 v 16:58 odesílatel kucerazdenek ***@***.***>
napsal:

**Comment by petrleocompel** [comment]
Modbus v nové verzi vypadá OK. Nahradím integraci ukázkou jak nakonfigurovat modbus.
Integrace bude označena jako `deprecated`/`discontinued` (zastaralá - ukončená)

**Comment by petrleocompel** [comment]
@kucerazdenek Ukázka modbus přidána do readme. (zatím nečteme vše ale funguje)
